robot/basestation/stream_capture.py

```diff
@@ -72,6 +77,61 @@ def stop_recording_feed(stream_url):
     print(message)
     return success, message
 
+def add_rotation(stream_url,rotation):
+    """
+    Add rotation value to the value of the key (stream_url) in active recordings
+    Converts the list to a tuple and handles errors
+    """
+    global active_recordings
+
+    stream_shortname = get_stream_shortname(stream_url)
+    success = True
+    message = "Successfully added rotation value"
+
+    try:
+        active_recordings[stream_shortname].append(rotation)
+        tuple(active_recordings[stream_shortname])
+    except (KeyError):
+        success = False
+        message = "Could not add rotation value of " + get_stream_shortname(stream_url)
+    return success, message
+
+
+def rotate_stream(filename, rotation):
+    """
+    Rotate stream so that it looks like the rotation of the GUI
+    The rotation value as seen on the GUI is defined as the following:
+    0: No rotation
+    1: 90 degrees clockwise rotation
+    2: 180 degrees rotation
+    3: 90 degrees counterclockwise rotation
+    """
+
+    rotation = int(rotation)
```

Comment:
I would recommend you pass the rotation as an integer instead of having the function 

`rotate_stream(string filename, string rotation)` it makes more sense you have it as
`rotate_stream(string filename, int rotation)`